### ISSUE TITLE
Tab min width is 1px larger than other Adwaita applications

### ISSUE THREAD

**Issue by kibisilicate** [rafaelmardojai/firefox-gnome-theme#459]
On firefox-gnome-theme v102, in `about:config` I found (through trial and error) that changing `browser.tabs.tabMinWidth` to "131" matches pixel perfect with other libadwaita applications.

After updating firefox-gnome-theme to the latest commit, I checked and it seems to be 1 pixel larger in width.

Changing the file `theme/parts/tabsbar.css` at line "399" from:
```
	--tab-min-width: 132px !important;
```
to:
```
	--tab-min-width: 131px !important;
```
makes the tab width identical with other libadwaita applications.

Before change:
![image](https://user-images.githubusercontent.com/48847651/195800030-5285b347-29b8-4632-8631-650a0b61d4be.png)

After change:
![image](https://user-images.githubusercontent.com/48847651/195800144-fbfbf20f-5988-4124-ab4b-14c988c41705.png)

The right end of the images shows it more clearly.

**Comment by rafaelmardojai** [comment]
Yup, the meter here is that `131px` gives me TOC 😅, but I will do that as a matter of pixel perfection.
